scripts/observers/config.py

```diff
@@ -80,7 +80,11 @@ def prepare_observer(shard_id, working_dir, config_folder):
 
 def main():
     load_dotenv()
-    working_dir = Path(Path.home() / os.getenv('WORKING_DIRECTORY'))
+    working_dir_var = os.getenv('WORKING_DIRECTORY')
+    if working_dir_var == "":
```

Comment:
duplicated code. might extract a function an use it in the both places

Reply:
done